account_consolidation/models/consolidation_profile.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import ValidationError
+
+
+class CompanyConsolidationProfile(models.Model):
+
+    _name = 'company.consolidation.profile'
+    _description = 'Subsidiary consolidation profile'
+    _order = 'sub_company_id'
+    _rec_name = 'sub_company_id'
+
+    @api.model
+    def _default_consolidation_percentage(self):
+        return 100
+
+    company_id = fields.Many2one(
+        comodel_name='res.company',
+        default=lambda self: self.env['res.company']._company_default_get(),
+        required=True
+    )
+    consolidation_percentage = fields.Float(
+        string='Consolidation percentage',
+        help='Define a percentage to consolidate this company (in percents)',
+        default=lambda self: self._default_consolidation_percentage(),
+        required=True
+    )
+    sub_company_id = fields.Many2one(
+        comodel_name='res.company',
+        required=True,
+        string='Subsidiary'
+    )
+
+    @api.constrains('consolidation_percentage')
+    def _check_consolidation_percentage(self):
+        for profile in self:
+            if (
+                    profile.consolidation_percentage < 0 or
```

Comment:
IMO, it's better to open a next line w/ an operator instead of leaving it hanging on a previous line